spec/helpers/administrate/application_helper_spec.rb

```diff
@@ -41,4 +41,43 @@
       end
     end
   end
+
+  describe "#svg_tag" do
+    subject { Nokogiri::HTML svg_tag('logo.svg',
+                                    svg_id: 'logo')}
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 47, 52 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.